src/core/modules/MilkyWay.cpp

```diff
@@ -83,6 +83,7 @@ void MilkyWay::init()
 	setFlagShow(conf->value("astro/flag_milky_way").toBool());
 	setIntensity(conf->value("astro/milky_way_intensity",1.).toDouble());
 	setSaturation(conf->value("astro/milky_way_saturation", 1.).toDouble());
+	setColor(Vec3f(conf->value("astro/milky_way_color", "1.0,1.0,1.0").toString()));
```

Comment:
Color for texture?

Reply:
Yes. It was always available. For the old texture it was even necessary to deviate from white to counteract the strong blue hue. While the current strong recommendation is to leave it white, the color can be used for personal preference or even artistic reasons. I am just unsure whether to even enable this setting (and similar for Zod. light, and, while I am at it, even the background color else only settable via TUI) in the GUI.